scripts/lib/setup_venv.py

```diff
@@ -34,6 +34,114 @@
     "libpq-dev",            # Needed by psycopg2
 ]
 
+def resolve_requirements_file(requirements_file):
+    # type: (str) -> List[str]
+    """
+    This function takes the path to a PIP requirement file as an
+    argument and returns a list of package names after resolving all
+    the included requirement files. For example:
```

Comment:
`expand_reqs` in `scripts/lib/hash_reqs.py` does the same thing.

```py
def expand_reqs(fpath):
    # type: (str) -> List[str]
    """
    Returns a sorted list of unique dependencies specified by the requirements file `fpath`.
    Removes comments from the output and recursively visits files specified inside `fpath`.
    `fpath` can be either an absolute path or a relative path.
    """
    ...
```

I think we should reuse that code.